tests/plugins/sqlite/SqlitePluginTest.cpp

```diff
@@ -706,6 +710,7 @@ void SqlitePluginTest::testGetSingleEvent()
         QCOMPARE(retrievedEvent[History::FieldMessageType], event[History::FieldMessageType]);
         QCOMPARE(retrievedEvent[History::FieldMessageStatus], event[History::FieldMessageStatus]);
         QCOMPARE(retrievedEvent[History::FieldReadTimestamp], event[History::FieldReadTimestamp]);
+        QCOMPARE(retrievedEvent[History::FieldSentTime], event[History::FieldSentTime]);
```

Comment:
Are there any tests which remain which test creating text events without a timestamp? From my reading of the code, that is still possible, right? Are you testing that you get the expected results out of such a transaction?

Reply:
yes, true need to add one